api/src/main/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceImpl.java

```diff
@@ -141,9 +149,17 @@ public DbUser syncComplianceTrainingStatusV2(DbUser dbUser, Agent agent)
                       accessModuleNameMapper::moduleFromBadge, determineCompletionTime));
 
       completionTimes.forEach(
-          (accessModuleName, timestamp) ->
-              accessModuleService.updateCompletionTime(
-                  dbUser, accessModuleName, timestamp.orElse(null)));
+          (accessModuleName, timestamp) -> {
+            var updatedUserAccessModule =
+                accessModuleService.updateCompletionTime(
+                    dbUser, accessModuleName, timestamp.orElse(null));
+            if (updatedUserAccessModule.getCompletionTime() != null) {
```

Comment:
Q: When will this if be false.

Reply:
I also found this confusing. Two common examples:
- The user hasn't completed any trainings and clicked "sync"
- The user completes the RT training but CT is not completed yet

Reply:
It will be helpful to add this info as comments

Reply:
Added 👍